src/searchForm/SearchForm.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react'
+import cc from 'classcat'
+
+import { color } from '_utils/branding'
+import SearchIcon from 'icon/searchIcon'
+import { CalendarIcon } from 'icon/calendarIcon'
+import Button, { ButtonStatus } from 'button'
+import { Bullet } from 'index'
+import { BulletTypes } from 'bullet'
+import { StandardSeat } from 'icon/standardSeat'
+import { DoubleArrowIcon } from 'icon/doubleArrowIcon'
+import Divider from 'divider'
+import { MediaSize, MediaSizeContext } from '_utils/mediaSizeProvider/MediaSizeProvider'
+
+export interface SearchFormProps {
+  className?: string
+  from?: string
+  to?: string
+  info?: string
+}
+
+export const separatorWidth: number = 14
+
+const SearchForm = ({ className }: SearchFormProps) => {
+  const mediaSize = useContext(MediaSizeContext)
+  return (
+    <div className={cc(['kirk-searchForm', className])}>
+      <div className="kirk-searchForm-from-container">
+        <div className="kirk-searchForm-from">
+          <Button status={ButtonStatus.UNSTYLED} className="kirk-searchForm-button">
+            <Bullet type={BulletTypes.SEARCH} className="kirk-bullet--searchForm" /> Leaving from
+          </Button>
+          {mediaSize === MediaSize.SMALL && <Divider />}
+        </div>
+        <div className="kirk-searchForm-invert">
+          <Button status={ButtonStatus.UNSTYLED}>
+            <DoubleArrowIcon iconColor={color.primary} />
+          </Button>
+          {mediaSize === MediaSize.SMALL && <Divider />}
+        </div>
+      </div>
+
+      <div className="kirk-searchForm-to">
+        <Button status={ButtonStatus.UNSTYLED} className="kirk-searchForm-button">
+          <Bullet type={BulletTypes.SEARCH} className="kirk-bullet--searchForm" /> Going to
+        </Button>
+        {mediaSize === MediaSize.SMALL && <Divider />}
+      </div>
+      <div className="kirk-searchForm-dateSeat-container">
+        <div className="kirk-searchForm-date">
+          <Button status={ButtonStatus.UNSTYLED} className="kirk-searchForm-button">
+            <CalendarIcon /> Today
+          </Button>
+        </div>
+        <div className="kirk-searchForm-seats">
```

Comment:
You can use common for `kirk-searchForm-date` and `kirk-searchForm-seats` and maybe none for the `Button`component

Reply:
I have some specific rules for date which I don't have for seats, so I'd rather keeping them apart.